javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -297,6 +297,146 @@ public void shouldCharSeqReturnNotNull() {
         assertThat((Iterable<Character>) charSeq("123")).isNotNull();
     }
 
+    @Test
+    public void shouldArrayWithSingleReturnNotNull() {
+        assertThat(array('1')).isNotNull();
+    }
+
+    @Test
+    public void shouldArrayWithVarArgReturnNotNull() {
+        assertThat(array('1', '2', '3')).isNotNull();
+    }
+
+    @Test
+    public void shouldArrayWithIterableReturnNotNull() {
+        assertThat(array(charSeq('1', '2', '3'))).isNotNull();
+    }
+
+    @Test
+    public void shouldArrayWithStreamReturnNotNull() {
+        assertThat(array(charSeq('1', '2', '3').toJavaList().stream())).isNotNull();
```

Comment:
Good idea, but how would we handle the difference between slang Stream and java stream?
`toStream` vs `stream` is not very desciptive in my opinion